sites/partners/cypress/integration/listing.spec.ts

```diff
@@ -102,21 +102,44 @@ describe("Listing Management Tests", () => {
     cy.get("a > .button").contains("Add Listing").click()
     cy.contains("New Listing")
 
+    // Test photo upload
     cy.getByTestId("add-photos-button").contains("Add Photo").click()
     cy.getByTestId("dropzone-input").attachFile(
       "cypress-automated-image-upload-071e2ab9-5a52-4f34-85f0-e41f696f4b96.jpeg",
       {
         subjectType: "drag-n-drop",
       }
     )
+    cy.getByTestId("drawer-photos-table").contains(
+      "cypress-automated-image-upload-071e2ab9-5a52-4f34-85f0-e41f696f4b96"
+    )
+    cy.getByTestId("listing-photo-uploaded").contains("Save").click()
+    cy.getByTestId("photos-table").contains(
+      "cypress-automated-image-upload-071e2ab9-5a52-4f34-85f0-e41f696f4b96"
+    )
+
+    cy.getByTestId("add-photos-button").contains("Edit Photos").click()
+    cy.getByTestId("dropzone-input").attachFile("test-listing.jpg", {
```

Comment:
can you use a similar naming scheme as the existing image with a uuid? the intention is to be able to search by that image name and have it be unique in cloudinary to do cleanup

Reply:
Oh sure, I didn't realize why it had the uuid and thought that was because of some other automated step.